datadog_checks_dev/datadog_checks/dev/tooling/commands/clean.py

```diff
@@ -53,12 +66,23 @@ def clean(ctx, check, compiled_only, all_matches, verbose):
             )
     else:
         path = os.getcwd()
+        if path == resolve_path(get_root()):
+            if force:
+                force_clean_root = True
+            else:
+                echo_warning("You are running this from the root of the integrations project")
+                echo_warning("Should we remove everything, including: ")
+                echo_warning(".cache, .coverage, .eggs, .pytest_cache, .tox, build, dist, and *.egg-info")
+                echo_warning("You can also use --force or -f to bypass this input")
+                input = raw_input()
+                if is_affirmative(input):
```

Comment:
Instead of adding this let's use http://click.pocoo.org/6/prompts/#confirmation-prompts

`click.confirm('Do you want to continue?', abort=True)`

Reply:
We don't want to abort otherwise though, we want it to clean the pyc files otherwise